Source/Core/Check.js

```diff
@@ -49,31 +49,35 @@ define([
      * Throws if test is greater than maximum
      *
      * @param {Number} test The value to test
+     * @param {String} name The name of the variable being tested
      * @param {Number} maximum The maximum allowed value
      * @exception {DeveloperError} test must not be greater than maximum
      * @exception {DeveloperError} Both test and maximum must be typeof 'number'
      */
-    Check.numeric.maximum = function (test, maximum) {
-        Check.typeOf.number(test);
-        Check.typeOf.number(maximum);
+    Check.numeric.maximum = function (test, name, maximum) {
+        Check.typeOf.number(test, 'test');
+        Check.typeOf.string(name, 'name');
```

Comment:
And looking at the how `Check.numeric` `minimum` and `maximum` are currently used, it doesn't look like they should actually check that the parameters are of the right types because this is currently always done before calling `Check.numeric.minimum` or `Check.numeric.maximum` anyway.